common/src/chain/config/builder.rs

```diff
@@ -107,14 +102,40 @@ impl ChainType {
                                 PerThousand::new(1).expect("must be valid"),
                                 PoSConsensusVersion::V1,
                             ),
-                        }),
+                        },
                     ),
                 ];
                 NetUpgrades::initialize(upgrades).expect("net upgrades")
             }
             ChainType::Signet => NetUpgrades::unit_tests(),
         }
     }
+
+    fn default_chainstate_upgrades(&self) -> NetUpgrades<ChainstateUpgrade> {
```

Comment:
Oh, please... let's stop using the word `default` altogether in consensus! :smile: